main/menu.ts

```diff
@@ -20,14 +20,28 @@ export default function setMenu(win: Electron.BrowserWindow) {
                     label: 'Undo',
                     accelerator: 'CmdOrCtrl+Z',
                     click: () => {
-                        win.webContents.send('nyaovim:exec-commands', ['undo']);
+                        const focusedWebContents = webContents.getFocusedWebContents();
+                        if (win.webContents === focusedWebContents) {
```

Comment:
Yes, but it will break if there are other types of webContents in the future (I know there are only nyaovim and the dev tool at this moment).